packages/block-library/src/navigation/edit/use-more-navigation-menu.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import useNavigationMenu from '../use-navigation-menu';
+import useNavigationEntities from '../use-navigation-entities';
+
+const useMoreNavigationMenu = () => {
```

Comment:
I'm wondering about the naming of this being specific to the More Menu. That appears tied to a specific piece of UI.

In the Navigation domain the concept here might be argued as being "do we have any menus (of any type)?". 

So could we consider making this hook

```suggestion
const useHasBlockOrClassicMenus = () => {
```

That might make more logical in other situations where we might end up needing to utlise the same concept.

Reply:
Or even just `useHasMenus`?